src/Page.js

```diff
@@ -927,7 +940,12 @@ class Page {
         .then(result => this.insertFooterFile(result))
         .then(result => Page.insertTemporaryStyles(result))
         .then(result => markbinder.resolveBaseUrl(result, fileConfig))
-        .then(result => markbinder.render(result, this.sourcePath, fileConfig))
+        .then((result) => {
+          // this.fixedHeader may be updated in previous step insertHeaderFile
+          // we need to reflect the update and pass it to the renderer, if necessary
+          fileConfig.fixedHeader = this.fixedHeader;
```

Comment:
should we pass `fileConfig` into `insertHeaderFile` instead? Best to keep related code close together, would remove the need for the comment as well